index.html

```diff
@@ -0,0 +1,121 @@
+        </li>
+        <li>Feature</li>
+        <li>How it works</li>
+        <li>Pricing</li>
+      </ul>
+    </nav>
+
+    <!-- Container Web Page -->
+
+    <div class="webapp-container">
+        <!-- EMAIL FORM -->
+
+      <div class="email-form">
+        <h2>Handcrafted, home made masterpieces</h2>
+        <input type="email" name="" placeholder="Enter your email address">
+        <button type="button">GET STARTED</button>
+      </div>
+
+      <!-- DESCRIPTIONS -->
+
+      <div class="description-container">
+        <ul>
+          <li class="list-description-container">
+            <img src="img/00.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Premium Materials</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+          <li class="list-description-container">
+            <img src="img/02.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Fast Shipping</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+          <li class="list-description-container">
+            <img src="img/01.png" alt="" srcset="">
+            <div class="descritpion-info">
+              <h3>Quality Assurance</h3>
+              <p>Our trombones use the shiniest brass which is sourced locally and this will increase the longevity of your purchase</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <!-- YouTube Link -->
+
+      <div class="youtube-link">
+        <iframe width="1280" height="504" src="https://www.youtube.com/embed/65L_VV_Yepk" title="Zoe Wees - Girls Like Us (Piano Version)" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      </div>
+
+      <!-- Pricing -->
+
+      <div class="pricing-container">
+        <div class="pricing-info">
+          <h3>TENOR TRAMBONE</h3>
+          <h4>$600</h4>
+          <p> Lorem ipsum<br>
+              Lorem ipsum<br>
+              Lorem ipsum dolor<br>
+              Lorem ipsum 3<br>
+          </p>
+          <button>SELECT</button>
+        </div>
+        <div class="pricing-info">
+          <h3>TENOR TRAMBONE</h3>
+          <h4>$600</h4>
+          <p> Lorem ipsum<br>
+              Lorem ipsum<br>
+              Lorem ipsum dolor<br>
+              Lorem ipsum<br>
+          </p>
+          <button>SELECT</button>
+        </div>
+        <div class="pricing-info">
+          <h3>TENOR TRAMBONE</h3>
+          <h4>$600</h4>
+          <p> Lorem ipsum<br>
+              Lorem ipsum<br>
+              Lorem ipsum dolor<br>
+              Lorem ipsum<br>
+          </p>
+          <button>SELECT</button>
+        </div>
+      </div>
+
```

Comment:
- I think it would be better if you could align these two sections well. 
- This will make your web page to look more beautiful

![Capture](https://user-images.githubusercontent.com/59427923/198931644-4954ecfa-0822-49b6-98a9-57dcd499cd70.PNG)